.github/ISSUE_TEMPLATE/default_issue.yml

```diff
@@ -0,0 +1,61 @@
+name: New Issue
```

Comment:
Why don't we call this "All Other Issues" or something like that, to make it really clear that it's the catch-all case.